specs/core/0_beacon-chain.md

```diff
@@ -2335,10 +2335,11 @@ def process_transfer(state: BeaconState, transfer: Transfer) -> None:
     assert get_balance(state, transfer.sender) >= max(transfer.amount, transfer.fee)
     # A transfer is valid in only one slot
     assert state.slot == transfer.slot
-    # Only withdrawn or not-yet-deposited accounts can transfer
+    # Only withdrawn, not-yet-deposited accounts, or the balance over MAX_EFFECTIVE_BALANCE can be transfered
     assert (
         get_current_epoch(state) >= state.validator_registry[transfer.sender].withdrawable_epoch or
-        state.validator_registry[transfer.sender].activation_epoch == FAR_FUTURE_EPOCH
+        state.validator_registry[transfer.sender].activation_epoch == FAR_FUTURE_EPOCH or
+        transfer.amount + transfer.fee >= get_balance(statetransfer.sender) - get_effective_balance(transfer.sender)
```

Comment:
Did you mean `transfer.amount + transfer.fee + MAX_EFFECTIVE_BALANCE <= get_balance(transfer.sender)`?

1) Changed `>=` to `<=`.
2) Change `get_effective_balance(transfer.sender)` to `MAX_EFFECTIVE_BALANCE`.
3) Fixed `statetransfer` typo.
4) Rearranged presentation to avoid underflows.